config/initializers/importers.rb

```diff
@@ -40,6 +40,16 @@
   satisfaction.range = ENV.fetch('GOOGLE_SATISFACTION_RANGE', 'A:K')
 end
 
+Rails.configuration.x.google_sessions.tap do |sessions|
+  sessions.service_account_email = ENV['GOOGLE_SESSIONS_EMAIL']
+  sessions.key_data = Base64.decode64(ENV.fetch('GOOGLE_SESSIONS_KEY', ''))
+  sessions.key_secret = ENV['GOOGLE_SESSIONS_SECRET']
+  sessions.sheets = {
+    sessions: ENV['GOOGLE_SESSIONS']
+  }
+  sessions.range = ENV.fetch('GOOGLE_SESSIONS_RANGE', 'A16:C16')
```

Comment:
Yeah, but no.

A) it should be 'Sessions!A16:C16'
B) it shouldn't have a default value
C) a named range in the file would  be better but not sure if that would survive running the report overnight